File: src/targets/ruby/native/fixtures/headers.rb
require 'uri'
require 'net/http'
require 'openssl'

url = URI("https://httpbin.org/headers")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true

request = Net::HTTP::Get.new(url)
request["accept"] = 'application/json'
request["x-foo"] = 'Bar'
<<<<<<< HEAD
request["x-bar"] = 'Foo'
=======
request["quoted-value"] = '"quoted" \'string\''
>>>>>>> upstream/master

response = http.request(request)
puts response.read_body

File: src/targets/node/axios/fixtures/headers.js
<<<<<<< HEAD
const axios = require('axios');

const options = {
  method: 'GET',
  url: 'https://httpbin.org/headers',
  headers: {accept: 'application/json', 'x-foo': 'Bar', 'x-bar': 'Foo'}
};

axios
  .request(options)
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.error(error);
  });
=======
const axios = require('axios').default;

const options = {
  method: 'GET',
  url: 'http://mockbin.com/har',
  headers: {
    accept: 'application/json',
    'x-foo': 'Bar',
    'quoted-value': '"quoted" \'string\''
  }
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master
